rdt-open-reader/src/covidIndonesia/assets/json.form/patient-registration-form-original.json

```diff
@@ -54,6 +54,13 @@
     "next_type": "submit",
     "submit_label": "SUBMIT",
     "fields": [
+      {
+        "key": "fields_required_hint",
+        "type": "label",
+        "text": "All fields marked with an asterisk (*) are required",
```

Comment:
The asterisk will not be red in this case so it may not be as clear. Could we change this to `Required fields` and make the label required so that the red asterisk appears?

After doing this it should read `Required fields(red asterisk)`

Reply:
done